src/semantic-ui/site/globals/site.variables

```diff
@@ -28,6 +28,8 @@
 @headlineBackground       : url("@{imagesFolder}/home-headline-background.jpg");
 @loginBackground          : url("@{imagesFolder}/login-background.png");
 @boSidebarWidth           : 230px;
+@linkColor                : #4183C4;
```

Comment:
this variable is already provided by the semantic-ui theme, so you are re-declaring it here